conanfile.py

```diff
@@ -15,8 +15,12 @@ class CrashpadConan(ConanFile):
     url = "https://github.com/bincrafters/conan-crashpad"
     topics = ("conan", "crash-reporting", "logging", "minidump", "crash")
     settings = "os", "compiler", "build_type", "arch"
-    options = {'linktime_optimization': [True, False]}
-    default_options = {"linktime_optimization": False}
+    options = {
+        "linktime_optimization": [True, False],
+        "force_embedded_zlib": [True, False]}
+    default_options = {
+        "linktime_optimization": False,
+        "force_embedded_zlib": False}
```

Comment:
What is the advantage of using the embedded zlib?

Reply:
Crashpad's zlib is configured (https://chromium.googlesource.com/crashpad/crashpad/+/refs/heads/master/third_party/zlib/BUILD.gn#17) to switch the zlib source based on the platform you're building on. That's not very transparent dependency management. In my project, I want to have a grip on where the libs are coming from and I prefer not pulling the system's current zlib version (see `zlib_source = "system"` in the link).

So I've put this option here. It doesn't do any harm, setting it `false` does exactly the same thing that before but `true` is a huge help when one needs to make sure where zlib comes from.

As I said, the best would be making crashpad's zlib to be a proper conan dependency but that's out of scope for me in this PR, it's already fixing a lot more than I first intended (which is good).

Reply:
Fine by me for now, though, in the long term, every single patch more is a burden 🙂